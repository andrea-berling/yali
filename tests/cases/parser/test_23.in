55 - 56 * 24 - 22
