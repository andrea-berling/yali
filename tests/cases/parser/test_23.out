(- (- 55.0 (* 56.0 24.0)) 22.0)
